packages/browser-ext/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Great find 👍 Can we add a short comment on top of these `.env` files to explain why we need this?

Reply:
Totally thought of that but spaced on the comment character for an .env file and moved on 😆